pkg/gofr/http/request.go

```diff
@@ -157,3 +160,22 @@ func (r *Request) bindForm(ptr any, isMultipart bool) error {
 
 	return nil
 }
+
+// bindBinary handles binding for binary/octet-stream content type.
+func (r *Request) bindBinary(raw interface{}) error {
+	// Ensure i is a pointer to a byte slice
```

Comment:
You didn't rename `i` here


```suggestion
	// Ensure raw is a pointer to a byte slice
```

Reply:
sure changed